### ISSUE TITLE
PluralSight Account Blocked

### ISSUE THREAD

**Issue by IsaacFlores1990** [rg3/youtube-dl#15516]
When I have already downloaded many videos an I try to resume the download my account gets blocked, because the random time you set is too small, I get the following error message:

WARNING: Unable to download JSON metadata: HTTP Error 403: Forbidden
[pluralsight] Waiting for 3 seconds to avoid throttling
ERROR: No video formats found;
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpww9373dy\build\youtube_dl\YoutubeDL.py", line 785, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpww9373dy\build\youtube_dl\extractor\common.py", line 440, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpww9373dy\build\youtube_dl\extractor\pluralsight.py", line 326, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpww9373dy\build\youtube_dl\extractor\common.py", line 1071, in _sort_formats
youtube_dl.utils.ExtractorError: No video formats found; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

As you can see only 3 seconds to avoid throttling is too small, I did call the command with --sleep-interval 60, and I did not have any issues in the past, until now when I had downloaded almost all videos if the curse and then tried to continue. I don't why for each video is downloaded the course JSON, I thinks this consults to their Web API added to the small time between queries cause the issue. How can I configure this time between queries, or can you get only once the JSON data?

**Comment by dstftw** [comment]
Carefully read [new issue template](https://github.com/rg3/youtube-dl/issues/new) and provide all requested information.
